lib/msal-common/apiReview/msal-common.api.md

```diff
@@ -3461,6 +3461,23 @@ export type RequestThumbprint = {
     shrOptions?: ShrOptions;
 };
 
+// Warning: (ae-missing-release-tag) "RequestValidator" is part of the package's API, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export class RequestValidator {
```

Comment:
Why is this class public?

Reply:
This change was auto-generated. The only change I made was export it in msal-common so it can be used in msal-node.

Reply:
I think this needs to be removed?

Reply:
It's exported in msal-common's index.ts. That makes it public.

Reply:
I don't think this is good enough reason. But in any case, there is no need to validate claims.